diff --git a/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle b/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
--- a/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
+++ b/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
@@ -92,7 +92,7 @@
     publishing {
       publications {
         release(MavenPublication) {
-          from components.release
+          from components.findByName("release")
         }
       }
       repositories {
